docs/guides/integration/docker.md

```diff
@@ -76,6 +76,15 @@ RUN uv sync --frozen
     in your repository to prevent it from being included in image builds. The project virtual
     environment is dependent on your local platform and should be created from scratch in the image.
 
+!!! note
```

Comment:
Sorry I should have noticed this in the first run. This is kind of a weird spot? Can we put this in a new section at the bottom or something? It feels quite prominent here and we're not talking about installing Python at all.